ray-on-gke/kuberay-tpu-webhook/main.go

```diff
@@ -107,6 +107,20 @@ func extractRayCluster(admissionReview *admissionv1.AdmissionReview) (*ray.RayCl
 	return &rayCluster, nil
 }
 
+// unmarshal rayjob from admission request
+func extractRayJob(admissionReview *admissionv1.AdmissionReview) (*ray.RayJob, error) {
```

Comment:
Do we have a test for this?

Reply:
Currently just manual tests - using `kubectl describe` on the RayJob to verify env vars are injected and viewing the webhook logs.